examples/3l-node/cli.rs

```diff
@@ -219,6 +229,12 @@ fn setup_editor(history_path: &Path) -> Editor<CommandHinter, DefaultHistory> {
         "payopeninvoice ",
     ));
 
+    hints.insert(CommandHint::new("listfailedswaps>", "listfailedswaps "));
+    hints.insert(CommandHint::new(
+        "refundfailedswap <swap address> <to address>",
+        "refundfailedswap ",
+    ));
```

Comment:
```suggestion
    hints.insert(CommandHint::new("listfailedswaps", "listfailedswaps"));
    hints.insert(CommandHint::new(
        "refundfailedswap <swap address> <to address>",
        "refundfailedswap",
    ));
```

Reply:
Thanks for spotting the extra `>` and ` ` in `listfailedswaps`. Removed in [4daf44f](https://github.com/getlipa/lipa-lightning-lib/pull/708/commits/4daf44fb38f3852ecaa1ed59e7cb584b902f8396).

The space after `refundfailedswap` is supposed to be there because there are parameters one has to provide to this command.